browser/resources/settings/email_aliases_page/content/remote_mapping_service.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { Alias, MappingService } from './types'
+// TODO: uncomment this once we have a real implementation
+//import { sendWithPromise } from 'chrome://resources/js/cr.js';
```

Comment:
as we're splitting this up anyway might be worth just removing the `sendWithPromise` stuff because it'll be as easy to implement with mojom :smile:

Reply:
done